change-notes/1.19/analysis-python.md

```diff
@@ -1,15 +1,15 @@
 # Improvements to Python analysis
 
-
 ## General improvements
 
-> Changes that affect alerts in many files or from many queries
-> For example, changes to file classification
+A new predicate `Stmt.getAnEntryNode()` has been added to make it easier to write reachability queries involving statements.
```

Comment:
This doesn't make as much sense out of context. This new predicate is necessary because of the changes to the CFG representation.

Reply:
Okay - that wasn't clear to me. I'll move it.